thinc/api.py

```diff
@@ -46,4 +46,62 @@
 from .layers import reduce_first, reduce_last, reduce_max, reduce_mean, reduce_sum
 
 
-__all__ = list(locals().keys())
+__all__ = [
```

Comment:
Does this need a `# fmt: off`?

Reply:
The formatting is as intended. Do we prefer a different one?

Reply:
`black` will reformat this, though.